addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -23,55 +81,36 @@ def test_region_CI_monkey_patch(self):
                 phonenumbers_patch._local_load_region,
                 "The code should not be monkey patched with phonenumbers > 8.12.32.",
             )
-        # check that you can load a new ivory coast phone number without error
-        parsed_phonenumber_1 = phonenumbers.parse("20 25/35-51 ", region="CI", keep_raw_input=True)
```

Comment:
Shouldn't you convert those to ``_assert_parsing_phonenumbers`` ? It is removed (and the other one is not converted)

Reply:
Right, this test is rather non-standard in the sense that it doesn't check for basic validity (using internal phonenumbers mechanics).

I added an option to `_assert_parsing_phonenumbers` so it is possible to skip the phonenumbers validity check.

However if number doesn't pass basic validity check (i.e. `phonenumbers.is_valid_number(parsed_phonenumber)`) That means that it won't get formatted correctly in odoo, so we probably never want to use that.

So currently I'm preserving what this test was doing before, however afaik this test isn't doing anything useful, unless I am unaware of a situation where we only need the country code to be detected correctly

Reply:
Would say the purpose was to parse ivory coast numbers :D if it is broken, maybe it deserves some check ?

Reply:
## Findings
So the [PR](https://github.com/odoo/odoo/pull/83045) and the issue that started CI (and all this patching) was only about matching newly created 10-digits numbers, to achieve that CI metadata was modified, specyfically possible lengts were set to 10-digits 
This is why tested numbers fails phonenumber internal [here] validation.

Why original creator went with 8-digit number then in his test? idk, maybe it was international number was mistaken with national, but I also see that at some in the history of `phonenumbers` both 8-digit and 10-digit numbers were accepted. That is no longer the case.

## What am gonna do about it?
Well I will just change the test data of Ivory Coast to use check **national** number with 10 digits